Makefile

```
@@ -98,6 +98,7 @@ endif
 
 # Change to `--upgrade-package <package-name>` to upgrade only a specific package
 MLRUN_UV_UPGRADE_FLAG ?= --upgrade
+COVERAGE?=false
```

Also please move to block above. Notice `# THIS BLOCK IS FOR VARIABLES USER MAY OVERRIDE` and `# THIS BLOCK IS FOR COMPUTED VARIABLES`
```suggestion
COVERAGE ?= false
```

---

I also think that `RUN_COVERAGE` or `INCLUDE_COVERAGE` could be a more fitting name